.cirrus.yml

```diff
@@ -5,21 +5,23 @@ low_scale_task:
     image: family/fedora-cloud-35
     platform: linux
     memory: 8G
+    use_static_ip: true
```

Comment:
Do we actually need this?

Reply:
No, that's a holdover from before I was using introspection to determine the proper IP. According to Cirrus's documentation, you can set `use_static_ip: true` and you are supposed to have a guaranteed IP address assigned. However, in practice this did not work.

Reply:
I guess, 'guaranteed IP address' is for how external services will see it, it doesn't mean that VM will have the same IP address, as that will obviously not work since they need to run more than one VM at the same time.

But they can nat everything to the same external IP, so you can add it to the allow list for your service, if CI needs to access that service, but you don't want to open it for the entire world.